packages/utils/addon/decorator.js

```diff
@@ -6,8 +6,14 @@ import {
   convertStage1ToStage2,
 } from './-private/class-field-descriptor';
 
+const IS_EDGE = /Edge\/\d/.test(window.navigator.userAgent);
```

Comment:
should this be wrapped with a FastBoot check?

Reply:
Yes